### PR TITLE
Simplify usage by updating to new default loop

### PR DESCRIPTION
**Pull request by clue**
This changeset simplifies usage by updating to use the new [default loop](https://github.com/reactphp/event-loop#loop).

Builds on top of https://github.com/reactphp/event-loop/pull/226, https://github.com/reactphp/event-loop/pull/232, https://github.com/reactphp/socket/pull/260 and https://github.com/reactphp/http/pull/410

### CODE REVIEW

**Comment by clue** [comment]
This is ready for review. I can merge once reviewed and the linked components have been tagged.

**Review by jsor — APPROVED** [review]

File: theme/homepage.html.twig [hunk 1]
```diff
@@ -53,15 +51,13 @@ $server = new React\Http\Server($loop, function (Psr\Http\Message\ServerRequestI
     );
 });
 
-$socket = new React\Socket\Server(8080, $loop);
+$socket = new React\Socket\Server('127.0.0.1:8080');
```
Comment (WyriHaximus) [inline theme/homepage.html.twig:54]: Might as well make it listen on all IP's

```suggestion
$socket = new React\Socket\Server('0.0.0.0:8080');
```

**Review by WyriHaximus — CHANGES_REQUESTED** [review]

File: theme/homepage.html.twig (hunk 1 above)
Comment (clue) [reply theme/homepage.html.twig:54]: No strong preference here tbh. I think recommending localhost listen address as a starting point is a more safe. Exposing this to the public internet requires only slightly more effort, but this is where things start to highly depend on use case / setup anyway, such as reverse proxy etc.. Additionally, the localhost address can also be used as is in URLs, which might make getting started slightly easier. WDYT?

**Review by WyriHaximus — APPROVED** [review]

File: theme/homepage.html.twig (hunk 1 above)
Comment (WyriHaximus) [reply theme/homepage.html.twig:54]: True, but IMHO we could better educate what values are often used in production. Any machine you startup should have a firewall enabled by default. I agree that it's safer, but we should also guide our users from start to production with our project and these little things help.

**Pull request merged**